### ISSUE TITLE
iOS: Remove autofocus after tap on disabled button

### ISSUE THREAD

**Issue by KarolDawidziuk** [ckeditor/ckeditor4#4766]
## Type of report

Task

## Provide description of the task

Disabled buttons fix on iOS it's no longer needed because since iOS8 this problem not occurs and it's changed behaves of editor compared to other systems. 

The change looks simple. We need just remove 2 piece of unnecessary code from:
1. https://github.com/ckeditor/ckeditor4/blob/master/plugins/button/plugin.js#L199-L201
2. https://github.com/ckeditor/ckeditor4/blob/master/plugins/dialog/plugin.js#L1151-L1153

## Other details

* Browser: ...
* OS: iOS
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by f1ames** [comment]
Blocked by #4767.

**Comment by f1ames** [comment]
It turns out it's not related nor blocked by #4767 as mentioned in https://github.com/ckeditor/ckeditor4/issues/4766#issuecomment-865871308.

**Comment by jacekbogdanski** [comment]
It seems like we are supporting iOS 6+ https://ckeditor.com/docs/ckeditor4/latest/guide/dev_browsers.html so let's check if the difference between maximize feature between Android and iOS is problematic (#4767) because we will introduce regression with this refactoring.

**Comment by jacekbogdanski** [comment]
We don't want to introduce regression for older iOS devices, so I'm closing this ticket.
